pkgs/by-name/do/double-entry-generator/package.nix

```diff
@@ -15,14 +16,30 @@ buildGoModule rec {
 
   vendorHash = "sha256-Xedva9oGteOnv3rP4Wo3sOHIPyuy2TYwkZV2BAuxY4M=";
 
-  postInstall = ''
-    rm $out/bin/hack
+  excludedPackages = [ "hack" ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=github.com/deb-sig/double-entry-generator/pkg/version.VERSION=${version}"
+    "-X=github.com/deb-sig/double-entry-generator/pkg/version.REPOROOT=github.com/deb-sig/double-entry-generator"
+    "-X=github.com/deb-sig/double-entry-generator/pkg/version.COMMIT=${src.rev}"
+  ];
+
+  doInstallCheck = true;
+  installCheckPhase = ''
```

Comment:
Oh, I forgot to add `runHook preInstallCheck` to the beginning and `runHook postInstallCheck` to the end.